app/src/main/java/com/nextcloud/talk/activities/CallActivity.java

```diff
@@ -1862,6 +1863,15 @@ private void processUsersInRoom(List<HashMap<String, Object>> users) {
                 } else {
                     oldSessions.add(participant.get("sessionId").toString());
                 }
+
+                // The property is "userId" when not using the external signaling server and "userid" when using it.
```

Comment:
Is that difference for a reason? Or simply history?

Reply:
As far as I know, simply history :-)

Reply:
For the record, after some investigation done for #2536 turns out that the comment (which was based on a comment in the WebUI code that has been there since forever) was not fully accurate (it is removed in #2536); although some external signaling messages actually use `userid` none of them were handled by the Android app, so `userId` could be always checked instead (and interestingly enough, after some archaeology it seems that `userId` could have been always checked in the WebUI too, but that is a different story :-P ).